insights/tests/client/connection/test_init_session.py

```diff
@@ -83,6 +83,14 @@ def test_get_no_proxy_env():
         connection.get_proxies()
     assert connection.proxies is None
 
+    with patch.dict(os_environ, {"HTTPS_PROXY": "env.proxy.example.com", "NO_PROXY": "redhat.com,example.com"}, clear=True):
```

Comment:
```suggestion
    with patch.dict(os_environ, {"HTTPS_PROXY": "env.proxy.example.com", "NO_PROXY": "example.com,redhat.com"}, clear=True):
```
The problem appears when getting values after a comma.

Reply:
I've added a series of new test cases including spaces and different ordering of the domain names.

Reply:
This looks great :smile: